src/main/resources/openapiv2.json

```diff
@@ -750,47 +936,58 @@
                     }
                 }
             },
-            "example": "{\n    \"partitions\": [\n        {\n            \"partition\": 1,\n            \"leader\": 1,\n            \"replicas\": [\n                {\n                    \"broker\": 1,\n                    \"leader\": true,\n                    \"in_sync\": true\n                },\n                {\n                    \"broker\": 2,\n                    \"leader\": false,\n                    \"in_sync\": false\n                }\n            ]\n        },\n        {\n            \"partition\": 2,\n            \"leader\": 2,\n            \"replicas\": [\n                {\n                    \"broker\": 1,\n                    \"leader\": false,\n                    \"in_sync\": true\n                },\n                {\n                    \"broker\": 2,\n                    \"leader\": true,\n                    \"in_sync\": false\n                }\n            ]\n        }\n    ]\n}"
+            "example": {
+                "partitions": [
+                    {
+                        "topic": "topic",
+                        "partition": 0
+                    },
+                    {
+                        "topic": "topic",
+                        "partition": 1
+                    }
+                ]
+            }
         },
-        "Offsets": {
-            "title": "Offsets",
+        "RecordsWithoutPartitions": {
```

Comment:
Are we using it? I cannot find it in the rest of the json file.

Reply:
`/topics/{topicname}/partitions/{partitionid}` uses it as response body.

Reply:
right! Clearly I have bad search skills on github :-) Btw I don't see any difference from ProducerRecordList even because you are defining the "records" field as a ProducerRecordList so why another type?